NCAR/main.yaml

```
@@ -149,7 +149,7 @@ sources:
     args:
       chunks: null
       consolidated: true
-      urlpath: /glade/derecho/scratch/digital-earths-hackathon/conus404/healpix/conus404_2020_*_hp{{ zoom }}.zarr
+      urlpath: /glade/derecho/scratch/digital-earths-hackathon/conus404/healpix/conus404_{{year}}_*_hp{{zoom}}.zarr
```

You could try to omit the `year` key word altogether and simply use a wildcard. Internally, this will make use of `xr.open_mfdataset()` and should give you a lazy view of the whole dataset:
```suggestion
      urlpath: /glade/derecho/scratch/digital-earths-hackathon/conus404/healpix/conus404_*_hp{{zoom}}.zarr
```

---

Oh yes, that's much better; thanks for the suggestion! Just done!